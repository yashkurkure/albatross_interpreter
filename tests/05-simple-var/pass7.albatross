var rXHG1Pi8a int := 4557;
var A2Gh67FFRA4ZvvVOOVXnh int := 65064;
var l1FGtKRUTrx0e8kEV4c int := 65043;
var kFa1KCwcmTCxfj9HpbTjE int := 5295;
var vLJ3xM int := 23289;
repeat (vLJ3xM-rXHG1Pi8a^!l1FGtKRUTrx0e8kEV4c&9175*54177>=27289^13717==16441>rXHG1Pi8a|l1FGtKRUTrx0e8kEV4c>20297<>kFa1KCwcmTCxfj9HpbTjE<>30761&&65524||l1FGtKRUTrx0e8kEV4c) {return 20928<26684==17831||64785%52343|36955>=vLJ3xM==5858<kFa1KCwcmTCxfj9HpbTjE<>l1FGtKRUTrx0e8kEV4c>=38915==49588==vLJ3xM>=60445%65463<=rXHG1Pi8a;}
if (50692<=5838|48172||kFa1KCwcmTCxfj9HpbTjE==A2Gh67FFRA4ZvvVOOVXnh&l1FGtKRUTrx0e8kEV4c>=63455&31098||kFa1KCwcmTCxfj9HpbTjE-5345-l1FGtKRUTrx0e8kEV4c-l1FGtKRUTrx0e8kEV4c-33963||vLJ3xM||15583) {repeat (32128>=41248>=26868&&21332*vLJ3xM&&14284+53844%10096-3033) {if (!54748) {while (15327==A2Gh67FFRA4ZvvVOOVXnh^55308+38080^kFa1KCwcmTCxfj9HpbTjE&&rXHG1Pi8a^rXHG1Pi8a<42827%vLJ3xM%vLJ3xM&2217<=A2Gh67FFRA4ZvvVOOVXnh-19006&59803-rXHG1Pi8a>=64897) {return vLJ3xM&&13199>=7854<>8856-!48749<>A2Gh67FFRA4ZvvVOOVXnh==60494==vLJ3xM&A2Gh67FFRA4ZvvVOOVXnh&7028&A2Gh67FFRA4ZvvVOOVXnh&rXHG1Pi8a&25404||27662;}}}}
rXHG1Pi8a:=!16975<=46317&&!16045+55931&&24136&&62086==36658<46245<5568<=63114&43471;
while (29180+l1FGtKRUTrx0e8kEV4c>=49441|48164>=vLJ3xM+24830<A2Gh67FFRA4ZvvVOOVXnh||rXHG1Pi8a-vLJ3xM) {repeat (!kFa1KCwcmTCxfj9HpbTjE+31182||vLJ3xM|3212<rXHG1Pi8a-vLJ3xM==30402<>rXHG1Pi8a^rXHG1Pi8a&A2Gh67FFRA4ZvvVOOVXnh>A2Gh67FFRA4ZvvVOOVXnh<>vLJ3xM|65231==19397) {repeat (rXHG1Pi8a<=31519||vLJ3xM-kFa1KCwcmTCxfj9HpbTjE&&!vLJ3xM+4193*27132>=9359+34992||23399==48389||l1FGtKRUTrx0e8kEV4c<>16780&13854) {return l1FGtKRUTrx0e8kEV4c==l1FGtKRUTrx0e8kEV4c==kFa1KCwcmTCxfj9HpbTjE<l1FGtKRUTrx0e8kEV4c<kFa1KCwcmTCxfj9HpbTjE==kFa1KCwcmTCxfj9HpbTjE<>58588>=32496||vLJ3xM<>kFa1KCwcmTCxfj9HpbTjE%65137<=!rXHG1Pi8a;}}}
repeat (l1FGtKRUTrx0e8kEV4c>42325>l1FGtKRUTrx0e8kEV4c+kFa1KCwcmTCxfj9HpbTjE||kFa1KCwcmTCxfj9HpbTjE^5858>vLJ3xM&29867*32689|5137-30155||62676) {if (59259&&vLJ3xM>23223&rXHG1Pi8a%l1FGtKRUTrx0e8kEV4c<rXHG1Pi8a^vLJ3xM-5565-35738<vLJ3xM>=32752||!vLJ3xM&&l1FGtKRUTrx0e8kEV4c==rXHG1Pi8a) {repeat (14424<>47689>=kFa1KCwcmTCxfj9HpbTjE&&49040<=24105%A2Gh67FFRA4ZvvVOOVXnh<>vLJ3xM^3256*vLJ3xM&&rXHG1Pi8a&&30155<>49808&rXHG1Pi8a<>27980+55349||l1FGtKRUTrx0e8kEV4c) {A2Gh67FFRA4ZvvVOOVXnh:=61955&&40354<>51056^57921||20726<>64650<>3285^!54047<>11347%28160||61731<=50136;}}}
