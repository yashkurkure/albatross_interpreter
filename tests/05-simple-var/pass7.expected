Variable declared "rXHG1Pi8a" type int
Variable declared "A2Gh67FFRA4ZvvVOOVXnh" type int
Variable declared "l1FGtKRUTrx0e8kEV4c" type int
Variable declared "kFa1KCwcmTCxfj9HpbTjE" type int
Variable declared "vLJ3xM" type int
Variable read "vLJ3xM" type int
Variable read "rXHG1Pi8a" type int
Variable read "l1FGtKRUTrx0e8kEV4c" type int
Variable read "rXHG1Pi8a" type int
Variable read "l1FGtKRUTrx0e8kEV4c" type int
Variable read "kFa1KCwcmTCxfj9HpbTjE" type int
Variable read "l1FGtKRUTrx0e8kEV4c" type int
Variable read "vLJ3xM" type int
Variable read "kFa1KCwcmTCxfj9HpbTjE" type int
Variable read "l1FGtKRUTrx0e8kEV4c" type int
Variable read "vLJ3xM" type int
Variable read "rXHG1Pi8a" type int
Variable read "kFa1KCwcmTCxfj9HpbTjE" type int
Variable read "A2Gh67FFRA4ZvvVOOVXnh" type int
Variable read "l1FGtKRUTrx0e8kEV4c" type int
Variable read "kFa1KCwcmTCxfj9HpbTjE" type int
Variable read "l1FGtKRUTrx0e8kEV4c" type int
Variable read "l1FGtKRUTrx0e8kEV4c" type int
Variable read "vLJ3xM" type int
Variable read "vLJ3xM" type int
Variable read "A2Gh67FFRA4ZvvVOOVXnh" type int
Variable read "kFa1KCwcmTCxfj9HpbTjE" type int
Variable read "rXHG1Pi8a" type int
Variable read "rXHG1Pi8a" type int
Variable read "vLJ3xM" type int
Variable read "vLJ3xM" type int
Variable read "A2Gh67FFRA4ZvvVOOVXnh" type int
Variable read "rXHG1Pi8a" type int
Variable read "vLJ3xM" type int
Variable read "A2Gh67FFRA4ZvvVOOVXnh" type int
Variable read "vLJ3xM" type int
Variable read "A2Gh67FFRA4ZvvVOOVXnh" type int
Variable read "A2Gh67FFRA4ZvvVOOVXnh" type int
Variable read "rXHG1Pi8a" type int
Variable written "rXHG1Pi8a" type int
Variable read "l1FGtKRUTrx0e8kEV4c" type int
Variable read "vLJ3xM" type int
Variable read "A2Gh67FFRA4ZvvVOOVXnh" type int
Variable read "rXHG1Pi8a" type int
Variable read "vLJ3xM" type int
Variable read "kFa1KCwcmTCxfj9HpbTjE" type int
Variable read "vLJ3xM" type int
Variable read "rXHG1Pi8a" type int
Variable read "vLJ3xM" type int
Variable read "rXHG1Pi8a" type int
Variable read "rXHG1Pi8a" type int
Variable read "A2Gh67FFRA4ZvvVOOVXnh" type int
Variable read "A2Gh67FFRA4ZvvVOOVXnh" type int
Variable read "vLJ3xM" type int
Variable read "rXHG1Pi8a" type int
Variable read "vLJ3xM" type int
Variable read "kFa1KCwcmTCxfj9HpbTjE" type int
Variable read "vLJ3xM" type int
Variable read "l1FGtKRUTrx0e8kEV4c" type int
Variable read "l1FGtKRUTrx0e8kEV4c" type int
Variable read "l1FGtKRUTrx0e8kEV4c" type int
Variable read "kFa1KCwcmTCxfj9HpbTjE" type int
Variable read "l1FGtKRUTrx0e8kEV4c" type int
Variable read "kFa1KCwcmTCxfj9HpbTjE" type int
Variable read "kFa1KCwcmTCxfj9HpbTjE" type int
Variable read "vLJ3xM" type int
Variable read "kFa1KCwcmTCxfj9HpbTjE" type int
Variable read "rXHG1Pi8a" type int
Variable read "l1FGtKRUTrx0e8kEV4c" type int
Variable read "l1FGtKRUTrx0e8kEV4c" type int
Variable read "kFa1KCwcmTCxfj9HpbTjE" type int
Variable read "kFa1KCwcmTCxfj9HpbTjE" type int
Variable read "vLJ3xM" type int
Variable read "vLJ3xM" type int
Variable read "rXHG1Pi8a" type int
Variable read "l1FGtKRUTrx0e8kEV4c" type int
Variable read "rXHG1Pi8a" type int
Variable read "vLJ3xM" type int
Variable read "vLJ3xM" type int
Variable read "vLJ3xM" type int
Variable read "l1FGtKRUTrx0e8kEV4c" type int
Variable read "rXHG1Pi8a" type int
Variable read "kFa1KCwcmTCxfj9HpbTjE" type int
Variable read "A2Gh67FFRA4ZvvVOOVXnh" type int
Variable read "vLJ3xM" type int
Variable read "vLJ3xM" type int
Variable read "rXHG1Pi8a" type int
Variable read "rXHG1Pi8a" type int
Variable read "l1FGtKRUTrx0e8kEV4c" type int
Variable written "A2Gh67FFRA4ZvvVOOVXnh" type int
